### PR TITLE
Ruby Capstone - Review

### PR DESCRIPTION
**Pull request by VitorGuedesMadeira**
### Catalog of my things (Group Project)
[project requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things.md)

**Learning objectives**
- Insert and query data in SQL. ✔️
- Use primary key & foreign key mechanisms for joining tables. ✔️
- Understand the different types of relationships between tables. ✔️
- Query multiple tables. ✔️
- Prepare complex queries that answer analytical questions. ✔️
- Run a program using the command line. ✔️
- Use Ruby syntax for basic programming operations. ✔️
- Apply Ruby best practices and language style guides in code. ✔️
- Describe the SOLID principles of OOP. ✔️
- Implement classes and objects in Ruby. ✔️
- Understand the four main principles of OOP.
- Implement encapsulation and inheritance with Ruby. ✔️
- Create a UML class diagram. ✔️
- Explain the difference between associations, aggregations, and composition in OOP. ✔️
- Set up associations between classes and objects. ✔️
- Store data in files. ✔️
- Build interactive console apps. ✔️
- Recognize the value of making equal contributions to group projects to produce the best outcome. ✔️ 

catalog_of_my_things

Here's the explanation of the project on this [sneak peak video](https://drive.google.com/file/d/1bbmFmHjFwZvMxpG_OWxNAHztFuvCXJcd/view?usp=sharing)
**_co-authors: Vitor G, Diego Y. and Muhammad A._**

tag

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    id INT GENERATED ALWAYS AS IDENTITY,
+    publish_date DATE NOT NULL,
+    archived BOOLEAN,
+    cover_state VARCHAR(150),
+    publisher VARCHAR(150),
+    book_color VARCHAR(50),
+    games_id INT,
+    PRIMARY KEY(id),
+    CONSTRAINT fk_authors FOREIGN KEY(books_id) REFERENCES authors(id)
+);
```
Comment (ShahierNashaat) [inline schema.sql:62]: - [ ] Great job working on the database but you have just missed one more table the `Label` table please create it as shown in the UML.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- All tests are passing ✔️
- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
- [ ] Kindly notice that you should record a video presenting the features of the project.
- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- All the members in the video should appear in the video and talk about the project.
- The video contains a highlight about a piece of the code you finds interesting or is proud of making.
- It's *recommended* that you use [Zoom](https://zoom.us/) recording features while on a call with your peers.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (VitorGuedesMadeira) [reply schema.sql:69]: We totally forgot about it! Already implemetend! Thanks for the feedback 😄

**Review by Munsa1 — APPROVED** [review]
Hi Team, 
congratulations-3

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**